supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/actors/GcpBatchAsyncBackendJobExecutionActor.scala

```diff
@@ -1067,6 +1068,16 @@ class GcpBatchAsyncBackendJobExecutionActor(override val standardParams: Standar
         throw new RuntimeException(s"handleExecutionSuccess not called with RunStatus.Success. Instead got $unknown")
     }
 
+  override def getStartAndEndTimes(runStatus: StandardAsyncRunState): Option[(OffsetDateTime, OffsetDateTime)] =
+    runStatus match {
+      case terminalRunStatus: TerminalRunStatus if terminalRunStatus.eventList.nonEmpty =>
+        val offsetDateTimes = terminalRunStatus.eventList.map(_.offsetDateTime)
+        Some((offsetDateTimes.min, offsetDateTimes.max))
+      case terminalRunStatus: TerminalRunStatus if terminalRunStatus.eventList.isEmpty => None
+      case unknown =>
+        throw new RuntimeException(s"getStartAndEndTimes not called with TerminalRunStatus. Instead got $unknown")
```

Comment:
IMO it's better to return `None` in this case (and the similar ones in other backends), we try to avoid throwing exceptions around.

Reply:
Good point! Done!